storage/cloud-client/acl_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2016 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gcloud import storage
+import gcloud.storage.acl
+import pytest
+
+import acl
+
+# Typically we'd use a @example.com address, but GCS requires a real Google
+# account.
+TEST_EMAIL = 'jonwayne@google.com'
+
+
+@pytest.fixture
+def test_bucket(cloud_config):
+    """Yields a bucket that resets its acl after the test completes."""
+    bucket = storage.Client().bucket(cloud_config.storage_bucket)
+    acl = gcloud.storage.acl.BucketACL(bucket)
+    object_default_acl = gcloud.storage.acl.DefaultObjectACL(bucket)
+    acl.reload()
+    object_default_acl.reload()
+    yield bucket
+    acl.save()
+    object_default_acl.save()
+
+
+@pytest.fixture
+def test_blob(cloud_config):
+    """Yields a blob that resets its acl after the test completes."""
+    bucket = storage.Client().bucket(cloud_config.storage_bucket)
+    blob = bucket.blob('storage_acl_test_sigil')
+    blob.upload_from_string('Hello, is it me you\'re looking for?')
```

Comment:
Heh. Might want to include a pointer back to this test, in case someone sees this object and is like, "wtf is this object"